liblangutil/Exceptions.cpp

```diff
@@ -26,7 +26,10 @@ using namespace std;
 using namespace solidity;
 using namespace solidity::langutil;
 
-Error::Error(Type _type, SourceLocation const& _location, string const& _description):
+ErrorId solidity::langutil::operator"" _error(unsigned long long _error) { return ErrorId{ _error }; }
```

Comment:
Isn't this defined in the header already?

Reply:
Now it is, c959341.